meetings/2025-05-14.md

```diff
@@ -0,0 +1,56 @@
+# Node.js  Next 10 Years team Meeting 2025-05-14
+
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=acpc60ZNXJk>
+* **GitHub Issue**: <https://github.com/nodejs/next-10/issues/345>
+
+## Present
+
+* Michael Dawson (Red Hat)
+* Marco Ippolito
+* Zach Tanner
+* Jakob
+* Erick Wendel
+
+## Agenda
+
+## Announcements
+
+* No announcement this week
+
+* Extracted from **next10-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/next-10
+
+* Funding for Node.js project - status updates [#335](https://github.com/nodejs/next-10/issues/335)
+  * stalled for now
+
+* 2025 Survey [#330](https://github.com/nodejs/next-10/issues/330)
+  * Marco - survey online, 1300 respondents so far.
+  * Agreement it will become banner to help promote
```

Comment:
The PR I could find is https://github.com/nodejs/nodejs.org/pull/7726. I don't see a subsequent one that makes it a banner nor chatter about making it one. Should we poke the website team?

Reply:
Not just badge because we decide that v18 EoL was more important